gatsby/src/docs/accounts/sso/azure-sso.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Azure Active Directory SSO'
+sidebar_order: 1
+---
+
+## Installation
+
+You'll need an [Office365 developer account](https://developer.microsoft.com/en-us/microsoft-365/dev-program) 
+before installing Azure Active Directory SSO.
```

Comment:
This can be deleted. It's for Sentry engineers who need to setup an AzureAD instance.